src/backend/core/tests/test_models_identities.py

```diff
@@ -93,21 +93,21 @@ def test_models_identities_email_unique_same_user():
 
 
 def test_models_identities_email_unique_different_users():
-    """The "email" field should not be unique among users."""
+    """The 'email' field should not be unique among users."""
     email = factories.IdentityFactory()
     factories.IdentityFactory(email=email.email)
```

Comment:
I know this is not the PR to raise this kind of questions but ... do we really want emails to not be unique among users ? 

The idea I imagine is to include mailing list ?

Reply:
Technically we don't need them to be unique as we rely on the sub... The email is just for sending emails. I didn't find any good reason why we should enforce this in the code. It could be troublesome in some scenarios and don't bring any good?